hatch.toml

```diff
@@ -15,7 +15,7 @@ scripts.clean = "git restore --source=HEAD --staged --worktree -- docs/release-n
 
 [envs.hatch-test]
 default-args = []
-features = ["test"]
+features = ["test", "dask-ml"]
```

Comment:
Why add this here?  I thought this PR specifically doesn't use an implementation?  Is this related to the other PR?

Reply:
I could also add the dask extra. It’s a bit unclear which features should be in the env specification.

I think if we’d have added the hatch config to `pyproject.toml`, we could maybe use local overrides in `hatch.toml` to specify extras we’re currently working on, but right now, there’s no good way to locally modify the set of extras used here I think.

Reply:
Hmmm, ok.  I think have a fully featured env is good, but I guess I am wondering what this has to do with the PR.  Am I misunderstanding this PR i.e., do we need something with `dask-ml` to make this?

Reply:
Ah, ok I see this is indeed duplicated from #3267.